CRM/Utils/PDF/Utils.php

```diff
@@ -83,9 +83,9 @@ public static function html2pdf($text, $fileName = 'civicrm.pdf', $output = FALS
     // Add a special region for the HTML header of PDF files:
     $pdfHeaderRegion = CRM_Core_Region::instance('export-document-header', FALSE);
     $htmlHeader = ($pdfHeaderRegion) ? $pdfHeaderRegion->render('', FALSE) : '';
-
+// height: 0 is a work-a-round for an extra empty page with wkhtmltopdf
     $html = "
-<html>
+<html style="height:0;">
```

Comment:
The test fails are because (a) you need to `\` backslash-escape the quotes since this whole block is inside quotes, and (b) the comment line needs to be indented 2 spaces.

Also by the way have you tried weasyprint since wkhtmltopdf is sort of abandoned?